contracts/test/DeployLightClientTestScript.s.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+
+import { LightClientTest as LCTest } from "./mocks/LightClientTest.sol";
+import { LightClient as LC } from "../src/LightClient.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract DeployLightClientTestScript is Script {
+    function run(uint32 numBlocksPerEpoch, uint64 numInitValidators, address owner)
+        external
+        returns (address payable proxyAddress, address admin, LC.LightClientState memory)
+    {
+        // TODO for a production deployment provide the right genesis state and value
+
+        string[] memory cmds = new string[](4);
+        cmds[0] = "diff-test";
+        cmds[1] = "mock-genesis";
+        cmds[2] = vm.toString(numBlocksPerEpoch);
+        cmds[3] = vm.toString(uint256(numInitValidators));
+
+        bytes memory result = vm.ffi(cmds);
+        (LC.LightClientState memory state,,) =
+            abi.decode(result, (LC.LightClientState, bytes32, bytes32));
+
+        return deployContract(state, numBlocksPerEpoch, owner);
+    }
+
+    function runDemo(uint32 numBlocksPerEpoch, address owner)
+        external
+        returns (address payable proxyAddress, address admin, LC.LightClientState memory)
+    {
+        string[] memory cmds = new string[](1);
+        cmds[0] = "gen-demo-genesis";
+
+        bytes memory result = vm.ffi(cmds);
+        LC.LightClientState memory state = abi.decode(result, (LC.LightClientState));
+
+        return deployContract(state, numBlocksPerEpoch, owner);
+    }
+
+    /// @notice deploys the impl, proxy & initializes the impl
+    /// @return proxyAddress The address of the proxy
+    /// @return admin The address of the admin
+
```

Comment:
Also returns the light client state.

Reply:
addressed here https://github.com/EspressoSystems/espresso-sequencer/commit/c6ff57f7d67653b9bc5bb3bf69bda7f5609f779e